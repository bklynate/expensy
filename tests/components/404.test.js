import React from 'react';
import { shallow } from 'enzyme';
import ErrorPage from './../../src/components/404';

describe('Error Page', () => {
  it('should display an error message if page not found', () => {
    const wrapper = shallow(<ErrorPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
