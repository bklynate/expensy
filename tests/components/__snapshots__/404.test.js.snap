// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page should display an error message if page not found 1`] = `
<div>
  <h1>
    404 Error - 
    <Link
      replace={false}
      to="/"
    >
      Go Home
    </Link>
  </h1>
</div>
`;
